package.json

```diff
@@ -16,14 +16,17 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint .",
+    "pretest": "node pretest.js",
     "test": "mocha --timeout 20000 --require test/init.js",
-    "posttest": "npm run lint"
+    "posttest": "node posttest.js && npm run lint"
```

Comment:
Created a story for our backlog to clean up the tests. https://github.com/strongloop-internal/scrum-apex/issues/268